src/terminal/parser/OutputStateMachineEngine.cpp

```diff
@@ -1162,7 +1172,7 @@ bool OutputStateMachineEngine::_VerifyDeviceAttributesParams(_In_reads_(cParams)
 // - Null terminates, then returns, the string that we've collected as part of the OSC string.
 // Arguments:
 // - ppwchTitle - a pointer to point to the Osc String to use as a title.
-// - pcchTitleLength - a pointer place the length of ppwchTitle into.
+// - pcchTitle  - a pointer place the length of ppwchTitle into.

```

Comment:
Yuck. We should file a follow on task to turn all these old-school pointer/size things into `std::wstring_view` and `std::string_view` throughout the parser/adapter...

Reply:
I supposed this was easier to build one char at a time ... I assume you'll file a follow up?

Reply:
#975 should take care of this follow-up :)